scripts/BTLH_data_exploration.R

```diff
@@ -146,6 +154,9 @@ BTLH_sites <- BTLH_HMN %>%
             N.Females = length(unique(Band.Number[Sex == "F"]))) %>% 
   arrange(N.Captures) %>% 
   as.data.frame
+# EZ: a few sites have the same location name except there's a 1 after it.
+# Can the data from these sites be combined?  (if, for example, the banding
+# station was moved just a little ways away but it's basically the same site)
 
```

Comment:
I had the same thought, and it is actually one of my questions for tomorrow. I think the sites with the 1 are very close, so merging them should be fine.